sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/DetectLanguageResult.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.ai.textanalytics.models;
+
+import com.azure.core.annotation.Fluent;
+
+import java.util.List;
+
+/**
+ * The DetectedLanguageResult model.
+ */
+@Fluent
```

Comment:
Do we expect people to set values on these results? It seems odd that this is not immutable since it is a `*Result`.

Reply:
updated to immutable